core/kazoo_voicemail/src/kvm_message.erl

```diff
@@ -337,6 +354,10 @@ do_copy(AccountId, FromId, OldBoxId, NBId, NBoxJ) ->
                          ,[FromDb, FromId, ToDb, ToId]
                          ),
             NotFound;
+        {'error', 'timeout'} ->
+            do_copy(AccountId, FromId, OldBoxId, NBId, NBoxJ, Loop - 1);
+        {'error', 'conflict'} ->
```

Comment:
I don't what would cause this but I know this was happened before. This is not resolving why conflict is happening.